### PR TITLE
Join From new seed

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
## Description

The data to be asked from lookup will now be fetched from seed section in constants. Kindly review that if the data is being sent to the seed, the corresponding handler is also verifying a seed.

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [x] small-scale cloud test (

### CODE REVIEW

File: src/libLookup/Synchronizer.cpp [hunk 1]
```diff
@@ -151,6 +151,19 @@ bool Synchronizer::FetchLatestDSBlocks(Lookup* lookup,
   return true;
 }
 
+bool Synchronizer::FetchLatestDSBlocksSeed(Lookup* lookup,
+                                           uint64_t currentBlockChainSize) {
+  if (LOOKUP_NODE_MODE) {
+    LOG_GENERAL(WARNING,
+                "Synchronizer::FetchLatestDSBlocks not expected to be "
```
Comment (sandipbhoir) [inline src/libLookup/Synchronizer.cpp]: incorrect function name in log message.

File: src/libLookup/Synchronizer.cpp [hunk 2]
```diff
@@ -164,6 +177,19 @@ bool Synchronizer::FetchLatestTxBlocks(Lookup* lookup,
   return true;
 }
 
+bool Synchronizer::FetchLatestTxBlockSeed(Lookup* lookup,
+                                          uint64_t currentBlockChainSize) {
+  if (LOOKUP_NODE_MODE) {
+    LOG_GENERAL(WARNING,
+                "Synchronizer::FetchLatestTxBlocks not expected to be "
```
Comment (sandipbhoir) [inline src/libLookup/Synchronizer.cpp]: incorrect function name in log message.

File: src/libLookup/Synchronizer.cpp [hunk 3]
```diff
@@ -176,6 +202,18 @@ bool Synchronizer::FetchLatestState(Lookup* lookup) {
   return true;
 }
 
+bool Synchronizer::FetchLatestStateSeed(Lookup* lookup) {
+  if (LOOKUP_NODE_MODE) {
+    LOG_GENERAL(WARNING,
+                "Synchronizer::FetchLatestState not expected to be called "
```
Comment (sandipbhoir) [inline src/libLookup/Synchronizer.cpp]: incorrect function name in log message.

File: src/libLookup/Lookup.cpp [hunk 4]
```diff
@@ -403,37 +419,26 @@ void Lookup::SendMessageToRandomLookupNode(const bytes& message) const {
...
+  {
+    lock_guard<mutex> g(m_mutexSeedNodes);
 
-  if (!Messenger::SetLookupGetSeedPeers(
-          getSeedPeersMessage, MessageOffset::BODY,
-          m_mediator.m_selfPeer.m_listenPortHost)) {
-    LOG_EPOCH(WARNING, to_string(m_mediator.m_currentEpochNum).c_str(),
-              "Messenger::SetLookupGetSeedPeers failed.");
-    return false;
+    for (auto node : m_seedNodes) {
```
Comment (ansnunez) [inline src/libLookup/Lookup.cpp]: `const auto& node` may be better

**Pull request merged**